packages/next/build/generate-build-id.ts

```diff
@@ -2,7 +2,11 @@ export async function generateBuildId (generate: () => string|null, fallback: ()
   let buildId = await generate()
   // If there's no buildId defined we'll fall back
   if (buildId === null) {
-    buildId = fallback()
+    // We also create a new buildId if it starts with `ad` to avoid false 
+    // positives with ad blockers
+    while (!buildId || buildId.includes('ad')) {
```

Comment:
Can this be a regex instead ? we want to avoid `AD` too

Reply:
On it.